zpages/code_string.go

```diff
@@ -1,26 +1,12 @@
-// Copyright 2017, OpenCensus Authors
```

Comment:
I presume the codegen is the one that erased the licenses eh?

Reply:
yeah, I think it's fine